src/main/java/net/einsteinsci/betterbeginnings/items/ItemRotisserie.java

```diff
@@ -1,10 +1,8 @@
 package net.einsteinsci.betterbeginnings.items;
 
 import net.einsteinsci.betterbeginnings.ModMain;
-import net.minecraft.item.Item;
-import net.minecraft.item.ItemStack;
 
-public class ItemRotisserie extends Item implements ICampfireUtensil
+public class ItemRotisserie extends ItemPan
```

Comment:
This change was restructuring to make the pan and rotisserie more extensible. All the changes in this file (and I think ItemPan.java, if any) need to be reverted.